src/monitor.rs

```diff
@@ -10,42 +10,11 @@
 //! [monitor_iter]: ./struct.AvailableMonitorsIter.html
```

Comment:
Let's remove references to `AvailableMonitorsIter` in this top-level comment as well! Looks good to me otherwise.